package.json

```diff
@@ -16,6 +22,13 @@
     "eslint-plugin-svelte3": "^3.2.0",
     "jest": "^27.2.5",
     "prettier": "^2.3.0",
+    "rollup": "^2.60.2",
+    "rollup-plugin-copy": "^3.4.0",
```

Comment:
I'm pretty against centralising this one - it's pretty bad, doesn't even support _watch_ :/ I'd rather we use `rollup-plugin-copy-watch` everywhere where we currently use _rollup-plugin-copy_ -> It's a drop in replacement anyway. That way, we also only install _rollup-plugin-copy_ once - within _rollup-plugin-copy-watch_. The latter does not need to be centralised, because `file:` dependencies just produce symlinks (at least on macOS/Linux).